docker/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+set -x
+
+VSPK_VERSION=$1
+SPEC_BRANCH=$2
+LANGUAGE_PLUGIN=$3
+
+[[ -z "$MONOLITHE_REPO" ]] && MONOLITHE_REPO="https://github.com/nuagenetworks/monolithe.git"
+[[ -z "$MONOLITHE_BRANCH" ]] && MONOLITHE_BRANCH="master"
+[[ -z "$SPEC_REPO" ]] && SPEC_REPO="https://github.com/nuagenetworks/vsd-api-specifications.git"
+
```

Comment:
```suggestion
[[ -z "$SPEC_TAG" ]] && SPEC_TAG="master"
[[ -z "$VSPKGENERATOR_REPO" ]] && VSPKGENERATOR_REPO="https://github.com/nuagenetworks/vspkgenerator.git"
[[ -z "$VSPKGENERATOR_BRANCH" ]] && VSPKGENERATOR_BRANCH="master"

```
Adding few items:
* Allow specifying which repo/branch of the vspkgenerator to use
* Allow specifying which spec tag/branch to use (for the use with different releases)